flox/core.py

```diff
@@ -35,6 +36,17 @@
 if TYPE_CHECKING:
     import dask.array.Array as DaskArray
 
+    T_Func = Union[str, Callable]
+    T_Funcs = Union[T_Func, Sequence[T_Func]]
+    T_Axis = int
+    T_Axiss = tuple[T_Axis, ...]  # TODO: var name grammar?
```

Comment:
```suggestion
    T_Axis = tuple[int, ...]  # TODO: var name grammar?
```

I think we can just use `int` there. Is there something we can copy (import) from numpy's typing?

Reply:
The single `T_Axis` case is used somewhere in core.py though and therefore I prefer using the single variable in the plural form of `T_Axis`.